apps/dashr-multipage-report/app.R

```diff
@@ -0,0 +1,1916 @@
+                    ax = 0, 
+                    ay = -40, 
+                    font = list(size = 8), 
+                    showarrow = FALSE, 
+                    text = "months after shock", 
+                    xref = "paper", 
+                    yref = "paper"
+                  )
+                ), 
+                autosize = TRUE, 
+                dragmode = "pan", 
+                height = "300",
+                width = "300",
+                hovermode = "closest", 
+                legend = list(
+                  x = 0.00805970149254, 
+                  y = 1.03496503497, 
+                  bgcolor = "rgba(255, 255, 255, 0)", 
+                  font = list(size = 9)
+                ), 
+                margin = list(
+                  r = 40, 
+                  t = 5, 
+                  b = -1, 
+                  l = 20, 
+                  pad = 0
+                ), 
+                paper_bgcolor = "rgba(0, 0, 0, 0)", 
+                plot_bgcolor = "rgba(0, 0, 0, 0)", 
+                showlegend = TRUE, 
+                xaxis = list(
+                  autorange = FALSE, 
+                  nticks = 19, 
+                  range = c(0.5, 18), 
+                  showgrid = FALSE, 
+                  tickfont = list(
+                    color = "rgb(68, 68, 68)", 
+                    size = "9"
+                  ), 
+                  ticks = "", 
+                  type = "linear", 
+                  zeroline = FALSE
+                ), 
+                yaxis = list(
+                  autorange = FALSE, 
+                  linecolor = "rgb(190, 191, 192)", 
+                  mirror = TRUE, 
+                  nticks = "9", 
+                  range = c(-0.4, 1.2), 
+                  showgrid = FALSE, 
+                  showline = TRUE, 
+                  side = "left", 
+                  tickfont = list(
+                    color = "rgb(68, 68, 68)", 
+                    size = 9
+                  ), 
+                  ticks = "outside", 
+                  ticksuffix = " ", 
+                  type = "linear", 
+                  zeroline = FALSE
+                ), 
+                yaxis2 = list(
+                  anchor = "x", 
+                  autorange = FALSE, 
+                  exponentformat = "e", 
+                  linecolor = "rgb(190, 191, 192)", 
+                  nticks = 9, 
+                  overlaying = "y", 
+                  range = c(-0.1, 0.3), 
+                  showgrid = FALSE, 
+                  side = "right", 
+                  tickfont = list(size = 9), 
+                  tickprefix = " ", 
+                  ticks = "outside", 
+                  type = "linear", 
+                  zerolinecolor = "rgb(190, 191, 192)"
+                )
+              )
+            )
+          )
+          
+        ), className = "six columns", style = list('height' = "250px")),
+        
+        htmlImg(src = "http://i.imgur.com/IJsVT9P.png", 
```

Comment:
Serve images in assets throughout code (basically get rid of all your imgur links). Right now, all of these end up showing up as broken images in the app:

![2019-06-27_10:41:31](https://user-images.githubusercontent.com/19624164/60275597-2994a780-98c8-11e9-958b-96aafcfb5020.png)